test/specs/elements/Button/ButtonGroup-test.js

```diff
@@ -34,4 +34,15 @@ describe('ButtonGroup', () => {
 
   common.propValueOnlyToClassName(ButtonGroup, 'color', SUI.COLORS)
   common.propValueOnlyToClassName(ButtonGroup, 'size', SUI.SIZES)
+
+  describe('buttons', () => {
```

Comment:
@levithomason Should we go with `buttons` or `items` prop?

Reply:
I think `buttons` for consistency.  The name should follow the sub-component or child components in this case:

```jsx
<Breadcrumb sections={...} />
<Feed events={...} />
<Modal actions={...} />
<Button.Group buttons={...} />
```

The exception is `<Card.Group items={...} itemsPerRow={...} />` which was an early exploratory API.  However, this will be replaced with standardized width props, #406.  It should be: `<Card.Group cards={...} widths={...} />`.